service/lombok.config

```diff
@@ -1,4 +0,0 @@
-config.stopBubbling = true
-lombok.extern.findbugs.addSuppressFBWarnings = true
-lombok.log.fieldName = LOGGER
```

Comment:
We use this in application for other PRs

Reply:
Gradle refused to work if this file exist.

Reply:
Just checked the other repo could you add to .gitignore?

Reply:
Done - new build available [here](http://ec2-3-9-114-173.eu-west-2.compute.amazonaws.com/blue/organizations/jenkins/integration-adaptor-gp2gp/detail/PR-10/13/pipeline/23)